scripts/build-artifacts.sh

```
@@ -6,6 +6,40 @@ get_mem_available_kb() {
     grep MemAvailable /proc/meminfo | awk '{print $2}'
 }
 
+copy_ignored_errors() {
+    local build_target_base="$1"
+    local target="$2"
+    
+    local present_working_dir="$(pwd)"
+    local config_file="$present_working_dir/configuration/$target.json"
+    if [ -f "$config_file" ]; then
+        local ignored_errors_path=$(jq -r '.ignoredErrors.path' "$config_file")
+        local ignored_errors_file="$present_working_dir/$ignored_errors_path/common-ignored-errors.json"
```

The `ignored_errors` directory is specified in each target's configuration file because it allows us to define multiple ignored errors JSON files. So instead of checking for just a single file named `common-ignored-errors.json`, we should do a check here for all `.JSON` files present in the `$ignored_errors_path` directory and copy ignored errors from all files. 

There should be 3 logical steps here:
1. First step is to look for all `.json` files in a `common-ignored-errors` directory. I know this doesn't exist right now but we'll need to create one and move the `common-ignored-errors.json` to that directory.
2. Second step is to check for all ignored files `.json` files in the directory defined in target's configuration file. 
3. And lastly we merge all of them together in one JSON and copy it over.